packages/front-end/components/GuidedGetStarted.tsx

```diff
@@ -0,0 +1,362 @@
+}) {
+  const [skippedSteps, setSkippedSteps] = useLocalStorage<{
+    [key: string]: boolean;
+  }>("onboarding-steps-skipped", {});
+  const [showVideo, setShowVideo] = useState(false);
+
+  const { metrics } = useDefinitions();
+  const settings = useOrgSettings();
+  const { datasources, mutateDefinitions } = useDefinitions();
+  const { apiCall } = useAuth();
+  const { update } = useUser();
+  const hasDataSource = datasources.length > 0;
+  const hasMetrics =
+    metrics.filter((m) => !m.id.match(/^met_sample/)).length > 0;
+  const hasExperiments =
+    experiments.filter((m) => !m.id.match(/^exp_sample/)).length > 0;
+  const allowImport = !(hasMetrics || hasExperiments) && !hasFileConfig();
+
+  const hasSampleExperiment = experiments.filter((m) =>
+    m.id.match(/^exp_sample/)
+  )[0];
+
+  const importSampleData = (source: string) => async () => {
+    const res = await apiCall<{
+      experiment: string;
+    }>(`/organization/sample-data`, {
+      method: "POST",
+    });
+    await mutateDefinitions();
+    await mutate();
+    track("Add Sample Data", {
+      source,
+    });
+    await router.push("/experiment/" + res.experiment);
+  };
+
+  const steps: Task[] = [
+    {
+      blackTitle: "Welcome to ",
+      purpleTitle: "GrowthBook!",
+      text:
+        "GrowthBook is a modular platform that enables teams to create feature flags and analyze experiment results. These features can be used together, or on their own - the choice is yours.",
+      completed: settings?.videoInstructionsViewed || false,
+      additionalCta: (
+        <Link href="/settings/team">
+          <a className="font-weight-bold">
+            Not an engineer? Invite a developer to get started.
+          </a>
+        </Link>
+      ),
+      render: (
+        <>
+          <div className={clsx(styles.playerWrapper, "col-lg-6")}>
+            {showVideo ? (
+              <ReactPlayer
+                className={clsx("mb-4")}
+                url="https://www.youtube.com/watch?v=1ASe3K46BEw"
+                playing={true}
+                controls={true}
+                width="100%"
+              />
+            ) : (
+              <img
+                role="button"
+                className={styles.videoPreview}
+                src="/images/intro-video-cover.png"
+                width={"100%"}
+                onClick={async () => {
+                  setShowVideo(true);
+                  await apiCall(`/organization`, {
+                    method: "PUT",
+                    body: JSON.stringify({
+                      settings: {
+                        videoInstructionsViewed: true,
+                      },
+                    }),
+                  });
+                  await update();
+                }}
+              />
+            )}
+          </div>
+          <button
+            onClick={() => setCurrentStep(currentStep + 1)}
+            className="btn btn-primary m-2"
+          >
+            Set up your SDK
```

Comment:
Thoughts on the verbiage below? I fear that if we do something like "Get Started: Set up your SDK" or "Start Your Account Setup" it might reduce click rates, where as, taking it one step at a time is less of an investment and someone is more likely to take action. 

<img width="1087" alt="Screen Shot 2022-09-07 at 9 13 16 AM" src="https://user-images.githubusercontent.com/75274610/188887055-e1bcb263-fc78-4b6b-8de5-6e3343b37781.png">